code/game/objects/structures/lattice.dm

```diff
@@ -30,8 +30,9 @@
 	for(var/obj/structure/lattice/LAT in loc)
 		if(LAT == src)
 			continue
-		stack_trace("multiple lattices found in ([loc.x], [loc.y], [loc.z])")
-		return INITIALIZE_HINT_QDEL
+		QDEL_IN(LAT, 0)
+		if(mapload)
+			stack_trace("multiple lattices found in ([loc.x], [loc.y], [loc.z])")
```

Comment:
Shouldn't the qdel be inside the mapload check here?

Reply:
No. Catwalk is subtype of lattice, when creating catwalk lattice must gone.